common/src/main/java/com/wynntils/models/raid/RaidModel.java

```diff
@@ -85,6 +94,19 @@ public void onTitle(TitleSetTextEvent event) {
     // so we have to check for the chat message
     @SubscribeEvent
     public void onChatMessage(ChatMessageReceivedEvent event) {
+        if (inBuffRoom()) {
+            Matcher matcher = RAID_CHOOSE_BUFF_PATTERN.matcher(
+                    ColorUtils.stripColors(event.getStyledText().getString()));
+            if (matcher.find()) {
+                String playerName = matcher.group(1);
+                String buff = matcher.group(2);
+
+                partyRaidBuffs
+                        .computeIfAbsent(playerName, k -> new ArrayList<>())
+                        .add(buff);
```

Comment:
What if here, instead of just storing the name and buff name you calculated the major ID here. So pass the buff group, eg "Bedrock III" into `RaidBuffs` and return "Heart of the Pack"

Reply:
I would be happy to change the structure such that we only track major Ids rather than every raid buff. The original logic was that they may be a useful feature in extending this tracking to support seeing the buffs your teammates have taken somewhere on screen as well as the buffs you've taken (similar to how lootrunning missions that you've taken are displayed)

Reply:
Ah if that is something you want to do in the future then by all means keep it this way